soc/arm/nxp_imx/mimx8mm6_m4/linker.ld

```diff
@@ -5,3 +5,14 @@
  */
 
 #include <arch/arm/aarch32/cortex_m/scripts/linker.ld>
+
+SECTIONS
+{
+#include <linker/rel-sections.ld>
+#ifdef CONFIG_OPENAMP_RSC_TABLE
+    SECTION_PROLOGUE(.resource_table,, SUBALIGN(8))
+    {
+        KEEP(*(.resource_table*))
+    } GROUP_LINK_IN(ROMABLE_REGION)
+#endif
+}
```

Comment:
Use tabs for indentation add a newline at the end of the file.

Reply:
The addition of this section has nothing to do with the IPM and should perhaps be moved in a separate commit.

I was asked to get this to work even when using a linker script generator.
See [this](https://github.com/zephyrproject-rtos/zephyr/pull/42440) PR for more details.